Src/Base/AMReX_GpuControl.H

```diff
@@ -84,6 +84,24 @@ namespace Gpu {
     inline bool inLaunchRegion() noexcept { return in_launch_region; }
     inline bool notInLaunchRegion() noexcept { return !in_launch_region; }
 
+    /**
+     * Enable/disable GPU kernel launches.
+     *
+     * \note Much of the original GPU execution patterns remain intact with this
+     * approach. Therefore it should not be used for comparing
+     * GPU to non-GPU performance.
+     *
+     * \code
+     *  Gpu::setLaunchRegion(0);
+     *
+     *  //...
+     *
+     *  Gpu::setLaunchRegion(1);
+     * \endcode
+     *
+     * Will disable the launching of GPU kernels between the calls.
```

Comment:
We want to be clear that it only switches to CPU for kernels launch with macros, but not functions like `amrex::ParallelFor`.

Reply:
The new description reads: 

This will only switch from GPU to CPU for kernels launched with macros. Functions like `amrex::ParallelFor` will be unaffected. Therefore it should not be used for comparing GPU to non-GPU performance or behavior.